packages/dev/s2-docs/pages/s2/Accordion.mdx

```diff
@@ -36,7 +36,7 @@ import {Accordion, Disclosure, DisclosureTitle, DisclosurePanel, type Key} from
 import {useState} from 'react';
 
 function Example() {
-  let [expandedKeys, setExpandedKeys] = useState(new Set<Key>(['system']));
+  let [expandedKeys, setExpandedKeys] = useState(new Set<Key>(['settings']));
```

Comment:
maybe leftover from a copy paste but noticed that our example didn't have key with "system" so updated it